### ISSUE TITLE
`terminal-notifier` specifies non-existant Ruby version

### ISSUE THREAD

**Issue by Gastove** [julienXX/terminal-notifier#241]
Hello!

I've just installed `terminal-notifier`; when I try to use it (in really any way), I get:

```
$ terminal-notifier -message "hi"
Failed to execute process '/usr/local/bin/terminal-notifier'. Reason:
The file '/usr/local/bin/terminal-notifier' specified the interpreter '/System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/bin/ruby', which is not an executable command.
```

On the one hand, this error message has a perfectly fair point -- the path it's looking for doesn't exist _at all_, and thus certainly isn't an executable command! On the other, I'm not sure... how it's calling in to that specific Ruby version at all. I have two different versions of Ruby installed -- 2.3 at `/usr/bin/ruby` and 2.4 at `/usr/local/bin/ruby`. 

I am not sufficiently familiar with Ruby to have a good intuition about this: is the right answer to find a way to install Ruby 2.0 at the path above? Or is there something in `terminal-notifier` that needs tweaking? 

Thanks very much!

# Please check everything that applies to your issue:
- [x] I looked in closed issues and it has not already been answered
- [ ] My issue appeared with a terminal-notifier update
- [ ] I'm using a tool that uses terminal-notifier (guard-notifier, node-notifier...)

# To help us debug your issue please include:
- the macOS version you use: mac0S High Sierra 10.13.2
- terminal-notifier version: 2.0.0
- how did you install terminal-notifier (binary download, homebrew, ruby gem...)?: `brew`
- step-by-step reproduction instructions:
  - Attempt to use `terminal-notifier` in any way. For instance: `$ terminal-notifier -message "hi"`

# Common issues and solution:
(I checked all these and they don't _seem_ to apply.)

**Comment by Gastove** [comment]
*a-HEM* Okay so, cool story: since teriminal-notifier was broken, I did a `brew uninstall`. Oddly, I kept getting `terminal-notifier` errors from a command-line tool that could optionally use it. Brew swore the package was gone -- but it turns out, a `terminal-notifier` script was still there under `/usr/local/bin`. I nuked that, re-installed with brew, and now everything works great. I have... ah. No idea how that happened, but it certainly isn't y'all's fault or problem.

Thanks for the package (which works great, especially when installed correctly ;P) and sorry for the issue-tracker noise ;P
